core/ecschema-editing/src/test/Merging/KindOfQuantityMerger.test.ts

```diff
@@ -0,0 +1,381 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { KindOfQuantity, Schema, SchemaContext } from "@itwin/ecschema-metadata";
+import { SchemaMerger } from "../../Merging/SchemaMerger";
+import { expect } from "chai";
+
+/* eslint-disable @typescript-eslint/naming-convention */
+
+describe("KindOfQuantity merge tests", () => {
+  let targetContext: SchemaContext;
+  let sourceContext: SchemaContext;
+
+  const sourceJson = {
+    $schema: "https://dev.bentley.com/json_schemas/ec/32/ecschema",
+    name: "SourceSchema",
+    version: "1.2.3",
+    alias: "source",
+  };
+  const targetJson = {
+    $schema: "https://dev.bentley.com/json_schemas/ec/32/ecschema",
+    name: "TargetSchema",
+    version: "1.0.0",
+    alias: "target",
+  };
+  const referenceJson = {
+    $schema: "https://dev.bentley.com/json_schemas/ec/32/ecschema",
+    name: "ReferenceSchema",
+    version: "1.2.0",
+    alias: "reference",
+    items: {
+      TestUnitSystem: {
+        schemaItemType: "UnitSystem",
+      },
+      TestPhenomenon: {
+        schemaItemType: "Phenomenon",
+        definition: "Test Phenomenon",
```

Comment:
The definition here should be a valid definition.  if this is a 'base' phenomenon then the definition should be the same as the name of the phenomenon.